olp-cpp-sdk-core/tests/utils/UtilsTest.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2023 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <gtest/gtest.h>
+
+#include <string>
+
+#include <olp/core/utils/Credentials.h>
+
+namespace {
+
+using UtilsTest = testing::Test;
+
+TEST(UtilsTest, Credentials) {
```

Comment:
Maybe split with SCOPED_TRACE? 
- Empty url
- Nothing to censor
- Censoring app_id, app_code